src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(element => element.sex === 'm');
+
+  const centuryMenDiedIn = men.filter(element =>
+    century ? Math.ceil(element.died / 100) === century : element);
+
+  const menAverage = centuryMenDiedIn.map(element =>
+    element.died - element.born);
+
+  return menAverage.reduce((a, b) => a + b) / menAverage.length;
```

Comment:
```suggestion
  return menAverage.reduce((a, b) => a + b, 0) / menAverage.length;
```
Again just good practice imo. Also on lines 53, 87